src/docs/framework-integration/angular.md

```diff
@@ -303,6 +317,92 @@ export const config: Config = {
 };
 ```
 
-### excludeComponents
+## FAQs
+
+### Do I have to use the `dist` output target?
```

Comment:
This section in the only new content for the FAQs. The rest was just moved to fall below the API section